<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MS_VS_GASClass</class>
 <widget class="QMainWindow" name="MS_VS_GASClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MS_VS_GAS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">	background-color: #171b22;
	color: #fff;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="openSecondTask">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #7289DA;
	selection-background-color: #7289DA;
	color: #fff;
	border-width: 2px;
	border-style: solid;
	border-radius: 10px;
	alignment: center;
	border-color: #0d1117;
}</string>
        </property>
        <property name="text">
         <string>2 Задание</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="openThirdTask">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #7289DA;
	selection-background-color: #7289DA;
	color: #fff;
	border-width: 2px;
	border-style: solid;
	border-radius: 10px;
	alignment: center;
	border-color: #0d1117;
}</string>
        </property>
        <property name="text">
         <string>3 Задание</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="openFirstTask">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #7289DA;
	selection-background-color: #7289DA;
	color: #fff;
	border-width: 2px;
	border-style: solid;
	border-radius: 10px;
	alignment: center;
	border-color: #0d1117;
}</string>
        </property>
        <property name="text">
         <string>1 Задание</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="aboutApp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #7289DA;
	selection-background-color: #7289DA;
	color: #fff;
	border-width: 2px;
	border-style: solid;
	border-radius: 10px;
	alignment: center;
	border-color: #0d1117;
}</string>
        </property>
        <property name="text">
         <string>О программе</string>
        </property>
        <property name="icon">
         <iconset resource="MS_VS_GAS.qrc">
          <normaloff>:/MS_VS_GAS/resources/about.png</normaloff>:/MS_VS_GAS/resources/about.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MS_VS_GAS.qrc"/>
 </resources>
 <connections/>
</ui>
